src/vm/memory/segments.zig

```diff
@@ -119,19 +121,55 @@ test "memory segment manager" {
     const relocatable_address_1 = memory_segment_manager.addSegment();
 
     // Check that the memory segment manager has one segment.
-    try std.testing.expect(memory_segment_manager.memory.num_segments == 1);
+    try expect(memory_segment_manager.memory.num_segments == 1);
```

Comment:
This is equivalent but there is a purpose built testing function for equality expectations, namely `std.testing.expectEqual`, see here for the full documentation: https://ziglang.org/documentation/0.11.0/#The-Testing-Namespace